Artifacts/linux-install-docker-container/linux_install_docker_container.sh

```diff
@@ -10,8 +12,20 @@
 #
 ###
 
-sudo apt-get -y update
-sudo apt-get -y install docker.io
+if [ -f /usr/bin/apt ] ; then
+    echo "Using APT package manager"
+
+    apt-get -y update
+    apt-get -y install docker.io
```

Comment:
docker.io starts the service and registers it to start on boot, whereas docker-engine does not.  docker.io is not available via Yum.